conjure_oxide/tests/generated_tests.rs

```diff
@@ -148,156 +148,195 @@ fn integration_test_inner(
             Default::default()
         };
 
-    // Stage 0: Compare the two methods of parsing
-    // skip if the field is set to true
-    // do not skip if it is unset, or if it is explicitly set to false
+    // Stage 0: Compare native parser with expected (unless skipped)
+    let mut model_native = None;
```

Comment:
```suggestion
    // Stage 0: Run native parser (unless skipped)
    let mut model_native = None;
```

Optional: we aren't actually doing the comparison here, but later